modules/gmc-agent/src/test/java/org/gridgain/action/controller/QueryActionsControllerWithParametersTest.java

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gridgain.action.controller;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Stream;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import net.minidev.json.JSONArray;
+import org.apache.ignite.IgniteException;
+import org.apache.ignite.lang.IgniteBiTuple;
+import org.gridgain.dto.action.Request;
+import org.gridgain.dto.action.query.QueryArgument;
+import org.junit.Before;
+import org.junit.Test;
+
+import static java.lang.String.format;
+import static org.gridgain.dto.action.ActionStatus.COMPLETED;
+
+/**
+ * Query actions controller with parameters test.
+ */
+public class QueryActionsControllerWithParametersTest extends AbstractActionControllerTest {
+    /** {@inheritDoc} */
+    @Before
+    @Override public void startup() throws Exception {
+        super.startup();
+
+        String insertAllTypesQry_1 = getInsertAllTypesQuery(
+            1,
+            false,
+            3,
+            10,
+            50,
+            new BigDecimal(900),
+            2d,
+            35f,
+            "12:34:57",
+            "2019-05-05",
+            "2019-05-05 12:34:57",
+            "char_1",
+            'a',
+            UUID.fromString("a-a-a-a-a")
+        );
+
+        String insertAllTypesQry_2 = getInsertAllTypesQuery(
+            2,
+            true,
+            30,
+            100,
+            500,
+            new BigDecimal(9000),
+            20d,
+            350f,
+            "13:40:25",
+            "2019-11-15",
+            "2019-11-15 13:40:25",
+            "char_2",
+            'b',
+            UUID.fromString("b-b-b-b-b")
+        );
+
+        Request initReq = new Request()
+            .setAction("QueryActions.executeSqlQuery")
+            .setId(UUID.randomUUID())
+            .setArgument(
+                new QueryArgument()
+                    .setQueryId("qry")
+                    .setQueryText(getAllTypeCreateQueryTable() + insertAllTypesQry_1 + insertAllTypesQry_2)
+                    .setPageSize(10)
+            );
+
+        executeAction(initReq, (r) -> r.getStatus() == COMPLETED);
```

Comment:
how about to assert returned result also?

Reply:
I think that we should not check the creation and insertion of results, because if this did not succeed, then we will not get the correct results with the following statements.